tap_github/__init__.py

```diff
@@ -2380,13 +2381,13 @@ def get_all_commit_files(schemas, repo_path,  state, mdata, start_date, gitLocal
                     commitQ.append(commit)
```

Comment:
Should also check newlyFetchedCommits up above on line 2378 too. Also, I'd do the initial declaration for newlyFetchCommits on line 2366 right before the loop. That way you don't have to assign it below on line 2408.